source/issue/page-title-too-simple/index.html.md

```diff
@@ -34,7 +34,7 @@ E.g. "Contact - Register your micropig" for a page which lets you "Enter your co
 
```

Comment:
I'm guessing it isn't that all page titles contain a keyword which is too vague, just the ones that are affected by this issue?